Plugin: unoconv文件转换

**插件名称：**

unoconv文件转换

**插件功能：**

基于unoconv的文件转换工具，支持输出pdf、doc、jpg等格式

**PyPI 项目名：**

nonebot-plugin-unoconv

**插件 import 包名：**

nonebot_plugin_unoconv

**插件项目仓库/主页链接：**

https://github.com/Zeta-qixi/nonebot-plugin-unoconv

**标签：**

[{"label":"文件转换","color":"#ea5252"},{"label":"pdf转换","color":"#ea5252"}]

# 📃 商店发布检查结果

> Plugin: unoconv文件转换

**⚠️ 在发布检查过程中，我们发现以下问题：**
<pre><code><li>⚠️ 插件加载测试未通过。<details><summary>测试输出</summary>插件 nonebot-plugin-unoconv 的信息如下：
    name         : nonebot-plugin-unoconv           
     version      : 0.5.6                            
     description  : nonebot_plugin about use unoconv 
    
    dependencies
     - nonebot-adapter-cqhttp &gt;=2.0.0b1
     - nonebot2 &gt;=2.0.0rc3
插件 nonebot_plugin_unoconv 加载出错：
03-16 01:02:38 [SUCCESS] nonebot | NoneBot is initializing...
03-16 01:02:38 [INFO] nonebot | Current Env: prod
03-16 01:02:38 [ERROR] nonebot | Failed to import &quot;nonebot_plugin_unoconv&quot;
Traceback (most recent call last):
  File &quot;/home/runner/work/nonebot2/nonebot2/plugin_test/runner.py&quot;, line 4, in &lt;module&gt;
    valid = load_plugin(&quot;nonebot_plugin_unoconv&quot;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
    return manager.load_plugin(module_path)
&gt; File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 141, in load_plugin
    module = importlib.import_module(name)
  File &quot;/usr/lib/python3.10/importlib/__init__.py&quot;, line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1050, in _gcd_import
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1027, in _find_and_load
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1006, in _find_and_load_unlocked
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 688, in _load_unlocked
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 233, in exec_module
    super().exec_module(module)
  File &quot;&lt;frozen importlib._bootstrap_external&gt;&quot;, line 883, in exec_module
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 241, in _call_with_frames_removed
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_unoconv/__init__.py&quot;, line 13, in &lt;module&gt;
    linux_install(&quot;unoconv&quot;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_unoconv/install.py&quot;, line 7, in linux_install
    if not subprocess.call([&quot;command&quot;, &quot;-v&quot;, pkg], stdout=subprocess.PIPE, stderr=subprocess.PIPE) == 0:
  File &quot;/usr/lib/python3.10/subprocess.py&quot;, line 345, in call
    with Popen(*popenargs, **kwargs) as p:
  File &quot;/usr/lib/python3.10/subprocess.py&quot;, line 969, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File &quot;/usr/lib/python3.10/subprocess.py&quot;, line 1845, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: &#x27;command&#x27;</details></li></code></pre>
<details><summary>详情</summary><pre><code><li>✅ 标签: 文件转换-#ea5252, pdf转换-#ea5252。</li><li>✅ 项目 <a href="https://github.com/Zeta-qixi/nonebot-plugin-unoconv">主页</a> 返回状态码 200。</li><li>✅ 包 <a href="https://pypi.org/project/nonebot-plugin-unoconv/">nonebot-plugin-unoconv</a> 已发布至 PyPI。</li></code></pre></details>

---

💡 如需修改信息，请直接修改 issue，机器人会自动更新检查结果。
💡 当插件加载测试失败时，请发布新版本后在当前页面下评论任意内容以触发测试。

💪 Powered by [NoneBot2 Publish Bot](https://github.com/nonebot/nonebot2-publish-bot)
<!-- PUBLISH_BOT -->

--- a/website/static/plugins.json
+++ b/website/static/plugins.json
@@ -4800,5 +4800,24 @@
       }
     ],
     "is_official": false
+  },
+  {
+    "module_name": "nonebot_plugin_unoconv",
+    "project_link": "nonebot-plugin-unoconv",
+    "name": "unoconv文件转换",
+    "desc": "基于unoconv的文件转换工具，支持输出pdf、doc、jpg等格式",
+    "author": "Zeta-qixi",
+    "homepage": "https://github.com/Zeta-qixi/nonebot-plugin-unoconv",
+    "tags": [
+      {
+        "label": "文件转换",
+        "color": "#ea5252"
+      },
+      {
+        "label": "pdf转换",
+        "color": "#ea5252"
+      }
+    ],
+    "is_official": false
   }
 ]

